app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <h2>Menu</h2>
+      <h4>
+        <ul>
+          <li>green salad</li>
+          <li>fresh lettuce, tomato and cheese</li>
+          <li>50</li>
+          <br>
+          <li>pasta carbonara</li>
+          <li>pasta with some delicious cheese sauce</li>
+          <li>102</li>
+          <br>
+          <li>ice cream</li>
+          <li>vanilla with chocolate sauce</li>
```

Comment:
You have hardcoded the values, that is not what we want, we want to bring the values from the db